Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Cultural flavorization

## Changelog:
- Added cultural titles for this cultures:
  - High Elves / Blood Elves
  - Night Elves
  - Nightborne / Highborne
  - Nagas
  - Humans: (+ specific for Kul'tiran, Solstice and Stromic)
  - Forsakens
  - Zandalaris
  - Drakkaris
  - Dragons: (+ specific for Red Dragons)
  - Mantids
  - Qirajis
  - Dwarfs: (+ specific for Bronzebeard, Wildhammer and Dark Iron)
  - Vrykuls: (+ specific for Frost Vrykuls)
  - Dryads
  - Tol'virs (+ specific when they're in evil religion)
  - Pandarens
  - Mogus
  - Goblins
  - Centaurs
  - Taurens
  - Nerubian
- Added princes titles for this cultures:
  - Dragons
  - Zandalaris
- Added specific titles for:
  - Qiraji king of Ahn'Qiraj
  - Mantids chancellor
  - Duchy of Ahn Qiraj
  - County of Grim'Batol
  - Barony of Grim'Batol

# How to test:
- Open the game
- Check all titles of all this cultures
- Check if a culture is forgotten


# If a culture does not have the corresponding title :
- Open the console (press ²)
- write `set_culture _yourculture_` for giving a new culture to your character. Example `set_culture lordaeronian` for become lordaeronian
- write `give_title _yourtitle_` for giving a new title to your character. Example `give_title e_arathor_empire` for giving you the Empire of Arathor
